src/core/Akka.API.Tests/CoreAPISpec.ApproveStreams.approved.txt

```diff
@@ -1760,8 +1760,8 @@ namespace Akka.Streams.Dsl
         public static Akka.Streams.SourceShape<T> Shape<T>(string name) { }
         public static Akka.Streams.Dsl.Source<T, Akka.NotUsed> Single<T>(T element) { }
         public static Akka.Streams.Dsl.Source<T, Akka.Actor.ICancelable> Tick<T>(System.TimeSpan initialDelay, System.TimeSpan interval, T tick) { }
-        public static Akka.Streams.Dsl.Source<TElem, Akka.NotUsed> Unfold<TState, TElem>(TState state, System.Func<TState, System.Tuple<TState, TElem>> unfold) { }
-        public static Akka.Streams.Dsl.Source<TElem, Akka.NotUsed> UnfoldAsync<TState, TElem>(TState state, System.Func<TState, System.Threading.Tasks.Task<System.Tuple<TState, TElem>>> unfoldAsync) { }
+        public static Akka.Streams.Dsl.Source<TElem, Akka.NotUsed> Unfold<TState, TElem>(TState state, System.Func<TState, Akka.Streams.Util.Option<System.Tuple<TState, TElem>>> unfold) { }
```

Comment:
This is an API change, but I know it's being done ultimately in service of https://github.com/akkadotnet/akka.net/pull/3973 - I'm fine with it.